src/js/components/projects/detail.tsx

```diff
@@ -58,7 +62,8 @@ const ProjectDetail = (
   const { project, projectSlug } = useFetchProjectIfMissing(props);
   const { epics } = useFetchEpicsIfMissing(project, props);
   const { orgs } = useFetchOrgsIfMissing({ project, props });
-
+  let tasks = useFetchTasksByProject(project);
```

Comment:
Since we only care about the `id` here, we could simplify what we're passing in:

```suggestion
  let tasks = useFetchTasksByProject(project?.id);
```